scipy/fftpack/tests/test_basic.py

```diff
@@ -324,6 +324,29 @@ def test_invalid_sizes(self):
         assert_raises(ValueError, rfft, [])
         assert_raises(ValueError, rfft, [[1,1],[2,2]], -5)
 
+    # See Issue #5790
```

Comment:
We usually refer to GitHub issues as gh-<N> from the code (I presume because that makes it easier to leave GitHub; SciPy was moved from SourceForge before).

Also, a one-sentence summary of the issue is in order.